templates/kafka/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Apache Kafka
+description: |
+  (Experimental) Kafka cluster
+version: 0.8.2-rancher1
+category: clustering
+release by: Raul Sanchez
+requires: rancher-0.47.0
```

Comment:
i saw this field in other packages in your offical repo. Do you have docs about the rules that you say??